matrix_sdk/src/verification_request.rs

```diff
@@ -32,7 +32,9 @@ impl VerificationRequest {
             let content = AnyMessageEventContent::KeyVerificationReady(content);
 
             self.client
-                .room_send(self.inner.room_id(), content, None)
+                .get_joined_room(self.inner.room_id())
+                .expect("Can't send a message to a room that isn't known to the store")
```

Comment:
I don't like this expect, a user might get kicked out of the room while a verification is going on. If the user accepts after the kick, without realizing that the kick happened we'll have a crash.